modules/@apostrophecms/area/ui/apos/apps/AposAreas.js

```diff
@@ -23,12 +23,31 @@ export default function() {
     const data = JSON.parse(el.getAttribute('data'));
     const fieldId = el.getAttribute('data-field-id');
     const choices = JSON.parse(el.getAttribute('data-choices'));
+    const renderings = {};
+    const _docId = data._docId;
+
+    for (const widgetEl of el.querySelectorAll('[data-apos-widget]')) {
```

Comment:
Does this work? You removed the only two places this attr appears in AposAreaWidget.